### PR TITLE
feat(helm-docs): Run helm-docs as part of the release automation

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
Fixes e.g. #146

### CODE REVIEW

**Comment by mjasion** [comment]
@0x2b3bfa0 maybe we could commit this to the PR 🤔 
https://github.com/iterative/helm-charts/blob/main/.github/workflows/test.yaml#L76-L80

https://github.com/iterative/helm-charts/blob/9d518ddf993837bbab0d0150815a67fee29438f3/.github/workflows/test.yaml#L76-L80

**Comment by mjasion** [comment]
@0x2b3bfa0 in #148 I created a workflow for automated helm-docs update

https://github.com/iterative/helm-charts/pull/148/files#diff-347d2e808a9367867a961b8037a015909424c2792aa650988042a6996ac64dd1

PTAL :-)

File: .github/workflows/helm-docs.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+jobs:
+  helm-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@b62528385c34dbc9f38e5f4225ac829252d1ea92
+        with:
+          app_id: ${{ secrets.PROJECT_ACTIONS_MANAGEMENT_APP_ID }}
+          private_key: ${{ secrets.PROJECT_ACTIONS_MANAGEMENT_APP_PEM }}
```
Comment (mjasion) [inline .github/workflows/helm-docs.yaml:24]: I right now used the GitHub App, but will propose some naming conventions for apps per repo or role 🤔

File: .github/workflows/helm-docs.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+      - name: Run helm-docs from go
+        working-directory: charts/studio/
+        run: |
+          go run github.com/norwoodj/helm-docs/...@v1.11.0
+
+      - name: Run helm-docs
+        working-directory: charts/studio/
+        run: |
+          docker run --pull always --rm --volume "$(pwd):/helm-docs" -u $(id -u) jnorwood/helm-docs:latest
```
Comment (mjasion) [inline .github/workflows/helm-docs.yaml:39]: No better way to compare times than on same pipeline ^^

**Review by mjasion — APPROVED** [review]

File: .github/workflows/helm-docs.yaml (hunk 2 above)
Comment (0x2b3bfa0) [reply .github/workflows/helm-docs.yaml:39]: Awesome! 😂

File: .github/workflows/helm-docs.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+      - uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: Helm-Docs update
+          file_pattern: charts/studio/README.md
```
Comment (0x2b3bfa0) [inline .github/workflows/helm-docs.yaml:44]: We probably don't need this, nor the token, because `cml pr` below does everything.

Comment (0x2b3bfa0) [reply .github/workflows/helm-docs.yaml:44]:
```suggestion
```

Comment (0x2b3bfa0) [reply .github/workflows/helm-docs.yaml:44]: My bad: reviewing two files at once 🤦🏼‍♂️

File: .github/workflows/helm-docs.yaml (hunk 2 above)
Comment (jesper7) [inline .github/workflows/helm-docs.yaml:39]: Why run `helm-docs` twice? Can we add a comment explaining why?

**Review by jesper7 — COMMENTED** [review]
Looks great! Just a minor comment

File: .github/workflows/helm-docs.yaml (hunk 2 above)
Comment (0x2b3bfa0) [reply .github/workflows/helm-docs.yaml:39]: For benchmarking; see https://github.com/iterative/helm-charts/pull/147#discussion_r1281995477

Comment (jesper7) [reply .github/workflows/helm-docs.yaml:39]: Do we still need the benchmark though?

Comment (0x2b3bfa0) [reply .github/workflows/helm-docs.yaml:39]: No, containers seem faster.

**Pull request merged**